lib/storage/clusterconfig/clusterconfig.go

```diff
@@ -145,6 +146,23 @@ func Marshal(config Interface, opts ...teleservices.MarshalOption) ([]byte, erro
 	return json.Marshal(config)
 }
 
+// ToUnknown returns this resource as a storage.UnknownResource
+func ToUnknown(config Interface) (*storage.UnknownResource, error) {
```

Comment:
I'm just curious: we already have `utils.ToUnknownResource` but it returns `teleservices.UnknownResource`. Is our `storage.UnknownResource` different?

Reply:
`teleservices.UnknownResource` does not properly serialize (it has a `Raw []byte` w/o json tags) and I need them to be serializable.